### DISCUSSION TITLE
files generated by erc/drc preflight options

### DISCUSSION THREAD

**Discussion by matthiesenj** [INTI-CMNB/KiBot#310] [Q&A] [answered]
I have noticed that enabling erc/drc in preflight causes a couple of txt and a filter file is generated.
These files end up in the path specified by -d when using kicad_auto:latest, and in the path specified by the "dir" directive in the global section when using kicad_auto:dev.

-Is this difference on purpose?
-Are these files considered temporary files?

**Comment by set-soft** [comment, marked as answer]
> -Is this difference on purpose?

As the CHANGELOG.md says *Now the global `dir` option also applies to the preflights, can be disabled using `use_dir_for_preflights`.* 
 for more information consult #292 (488f2dcbc27d1d457c29a98a552256a55fc10ff0)

> -Are these files considered temporary files?

Nope, these are the KiCad outputs, they could contain more information than the one collected by KiAuto/KiBot.
